src/ResourceManager/DevTestLabs/Commands.DevTestLabs/Commands/Set/SetAzureRmDtlVMsPerLabPolicy.cs

```diff
@@ -0,0 +1,57 @@
+﻿// ----------------------------------------------------------------------------------
+//
+// Copyright Microsoft Corporation
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// ----------------------------------------------------------------------------------
+
+using Microsoft.Azure.Management.DevTestLabs;
+using Microsoft.Azure.Management.DevTestLabs.Models;
+using System;
+using System.Management.Automation;
+
+namespace Microsoft.Azure.Commands.DevTestLabs
+{
+    [Cmdlet(VerbsCommon.Set, "AzureRmDtlVMsPerLabPolicy", HelpUri = Constants.DevTestLabsHelpUri, DefaultParameterSetName = ParameterSetEnable)]
+    public class SetAzureRmDtlVMsPerLabPolicy : DtlPolicyCmdletBase
+    {
+        #region Input Parameter Definitions
+
+        /// <summary>
+        /// Maximum number of virtual machines.
+        /// </summary>
+        [Parameter(Mandatory = true,
```

Comment:
Does it make sense to have a single set cmdlets and set the scope of maxVMs to peruser or perlab?